### PR TITLE
perf: remove customized items and add the relevant usage instead

### PR DESCRIPTION
**Pull request by msclock**

### CODE REVIEW

File: docs/topics/overlays.md [hunk 1]
```diff
@@ -26,3 +26,13 @@ References:
...
+
+## Overlay of a custom chinaload toolchain
+
+>  a toolchain file is selected from `scripts/toolchains/` appropriate to the platform by default.
+
+To use a custom chianload toolchain, a directory `cmake/vcpkg/scripts/toolchains` has been created in the root of the scaffold.
+
+References:
+
+- [What is a chinaload toolchain in vcpkg?](https://learn.microsoft.com/en-us/vcpkg/users/triplets#vcpkg_chainload_toolchain_file)
```
Comment (huxuan) [inline docs/topics/overlays.md:38]: s/chinaload/chainload/

File: template/[% if docs_type == 'sphinx' %]docs[% endif %]/[% if repo_name == 'ss-cpp' %]topics[% endif %]/overlays.md [hunk 2]
```diff
@@ -26,3 +26,13 @@ References:
 
 - <https://learn.microsoft.com/en-us/vcpkg/concepts/overlay-ports>
 - [Why overlay ports? Install locally modified package](https://learn.microsoft.com/en-us/vcpkg/consume/install-locally-modified-package)
+
+## Overlay of a custom chinaload toolchain
```
Comment (huxuan) [inline template/[% if docs_type == 'sphinx' %]docs[% endif %]/[% if repo_name == 'ss-cpp' %]topics[% endif %]/overlays.md:30]: same typo here, may need to check globally.

**Review by huxuan — COMMENTED** [review]
Minor typo comments

File: docs/topics/overlays.md (hunk 1 above)
Comment (msclock) [reply docs/topics/overlays.md:38]: Thanks pointing out🫤

File: template/[% if docs_type == 'sphinx' %]docs[% endif %]/[% if repo_name == 'ss-cpp' %]topics[% endif %]/overlays.md (hunk 2 above)
Comment (msclock) [reply template/[% if docs_type == 'sphinx' %]docs[% endif %]/[% if repo_name == 'ss-cpp' %]topics[% endif %]/overlays.md]: I have checked codebase globally, and it only has these two.

**Review by huxuan — APPROVED** [review]

**Pull request merged**